.github/workflows/cla.yml

```
@@ -36,3 +36,4 @@ jobs:
                   remote-organization-name: 'Expensify'
                   remote-repository-name: 'CLA'
                   lock-pullrequest-aftermerge: false
+                  allowlist: dependabot
```

let's also add synk-bot since we know it too is always creating PRs https://github.com/Expensify/expensify-common/pull/372.

```suggestion
                  allowlist: dependabot,snyk-bot
```

usage for this key https://github.com/cla-assistant/github-action#5-users-and-bots-in-allowlist